libipl/p10/ipl0.C

```diff
@@ -435,7 +562,16 @@ static int ipl_proc_clock_test(void)
 			rc++;
 		}
 
-		ipl_error_callback((fapirc == fapi2::FAPI2_RC_SUCCESS) ? IPL_ERR_OK : IPL_ERR_HWP);
+		if(fapirc == fapi2::FAPI2_RC_SUCCESS){
+			ipl_error_callback(IPL_ERR_HWP);
```

Comment:
should the condition at line 565 be != ?

Reply:
yes. basically i am planning to remove this change, since this is not good approach